inspirehep/modules/records/__init__.py

```diff
@@ -23,3 +23,5 @@
 """Data model package."""
 
 from __future__ import absolute_import, division, print_function
+
+from .ext import InspireRecords  # noqa: F401
```

Comment:
Is this needed? you can use the full path to ext (like the utils extension).

I'd prefer that (no strong opinion though).